tgui/packages/tgui/interfaces/MedicalRecords/RecordView.tsx

```diff
@@ -111,6 +114,58 @@ export const MedicalRecordView = (props, context) => {
                 text={blood_type}
               />
             </LabeledList.Item>
+            <LabeledList.Item
+              buttons={physical_statuses.map((button, index) => {
+                const isSelected = button === physical_status;
+                return (
+                  <Button
+                    color={isSelected ? PHYSICALSTATUS2COLOR[button] : 'grey'}
+                    icon={isSelected ? 'check' : ''}
+                    key={index}
+                    onClick={() =>
+                      act('set_physical_status', {
+                        crew_ref: crew_ref,
+                        physical_status: button,
+                      })
+                    }
+                    width={!isSelected ? '3.0rem' : 3.0}
+                    tooltip={PHYSICALSTATUS2DESC[button] || ''}
+                    tooltipPosition="bottom-start">
+                    {button[0]}
+                  </Button>
+                );
+              })}
+              label="Physical Status">
+              <Box color={PHYSICALSTATUS2COLOR[physical_status]}>
+                {physical_status}
+              </Box>
+            </LabeledList.Item>
+            <LabeledList.Item
+              buttons={mental_statuses.map((button, index) => {
+                const isSelected = button === mental_status;
+                return (
+                  <Button
+                    color={isSelected ? MENTALSTATUS2COLOR[button] : 'grey'}
+                    icon={isSelected ? 'check' : ''}
```

Comment:
I don't know why, but the buttons still feel a bit empty/unhelpful with just the single-letter... What we could do is set up a common `MENTALSTATUS2ICON` that will output a correspondent fontawesome icon that matches up to what it means, like:

Deceased could be https://fontawesome.com/icons/skull-crossbones?s=solid&f=classic (you would put this in the constant as `skull-crossbones`)
Active could be `face-smile`
Observe could be `camera`(?)

i think stuff like this would just make it all feel a bit more cohesive/better from a UX standpoint, getting an idea of what it might mean from the icon. this might be a bit hard so let me know if you're unable to complete it
(note: we can only use the free 6.3.0 (iirc) icons in FA)

Reply:
Per your suggestion, I updated the buttons to include both the letter and a relevant symbol relating to each status. Additionally, I've updated the image in the PR body to showcase this new look. Let me know if there's anything else that needs adjustment.